### ISSUE TITLE
Depend on Ramda instead of LoDash

### ISSUE THREAD

**Issue by cgarciae** [loop-recur/lambdajs#6]
Lamda currently depends on _LoDash_ for the functions `compose` and `curry`, I propose to depend on **Ramda** instead. The reason: string representation of functions.

Currently the string representation of a higher order function is an anonymous function with some code unrelated to the function of interest. Example:

```
console.log (add (1)); // ==>
/*
function bound() {
    var thisBinding = isBind ? thisArg : this;
    if (partialArgs) {
      var args = slice(partialArgs);
...LOTS OF MORE UGLY STUFF...
*/
```

With Ramda instead you get:

```
console.log (add (1)); // ==>
/*
function ( x, y ) {
      return x + y;
    } 
/*
```

**Comment by DrBoolean** [comment]
And the fn.toString() trick comes full circle :)

I'm down, but i don't have time to do this for a little bit. Definitely will pull though.

**Comment by cgarciae** [comment]
Yeah, heard you mentioned on this trick. It wouldn't be right not to have it on your library ;)
